### PR TITLE
revise food instructions

### PR DESCRIPTION
**Pull request by aryarm**
this pr clarifies some things that have come up since we started the food doc

it also tries to help calibrate the formula that we use for deciding the quantity of food to order

### CODE REVIEW

**Comment by aryarm** [comment]
@gymreklab, instead of linking to the GDrive, I just linked to your contact info on the lab website, which I also updated to indicate your new room at FAH. I hope that's ok?

**Review by yli091230 — APPROVED** [review]

**Comment by aryarm** [comment]
thanks for reviewing, @yli091230! I'll just wait for approval from @gymreklab for the changes to her contact info

**Comment by aryarm** [comment]
I got verbal confirmation from melissa that the changes to her contact info were ok, so I'm going to merge this now!

**Pull request merged**
